understanding/21/non-text-contrast.html

```diff
@@ -11,20 +11,22 @@ <h1>Understanding Non-text Contrast</h1>
 			<h2>Intent</h2>
 
 
-			<p>The intent of this Success Criterion is to ensure that active user interface components (i.e., controls) and meaningful graphics are distinguishable by people with moderately low vision. The requirements and rationale are similar to those for large text in <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>.</p>
+			<p>The intent of this Success Criterion is to ensure that active user interface components (i.e., controls) and meaningful graphics are distinguishable by people with moderately low vision who do not use contrast-enhancing assistive technology. The requirements and rationale are similar to those for large text in <a href="contrast-minimum">1.4.3 Contrast (Minimum)</a>.</p>
```

Comment:
I don't think this change is relevant, even if you _do_ use a contrast enhancing technology many graphics won't increase in contrast based on that usage.